cmd/manager/main.go

```diff
@@ -111,6 +111,9 @@ func main() {
 	startPoolRange := loadMacAddressFromEnvVar(poolmanager.StartPoolRangeEnv)
 	endPoolRange := loadMacAddressFromEnvVar(poolmanager.EndPoolRangeEnv)
 
+	err = webhook.CreateOwnerRefForMutatingWebhook(clientset)
```

Comment:
please add inline documentation saying why we need that

Reply:
done